scripts/engine.js

```diff
@@ -18,8 +18,23 @@ function loadEvenOdd() {
   document.getElementById("parkExplanation").innerHTML = parkingDeets.get('parkExplanation');
 }
 
+function loadLights() {
+  const now = new Date();
```

Comment:
Oh, I wonder if the `const now` from `loadEvenOdd()` should just be brought up out of the function so it can be used in both of these functions?

Reply:
Yes. I moved `const now = new Date();` from each function to the function that calls `loadEvenOdd()` and `loadLights()`. `now` is then passed as a parameter to each function.